### PR TITLE
synopsysctl stop will print more informative message if application

### PR DESCRIPTION
**Pull request by rrati**
doesn't appear to be running #752

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/772#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_stop.go [hunk 1]
```diff
@@ -55,9 +57,12 @@ var stopAlertCmd = &cobra.Command{
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
 		errors := []error{}
-		for _, altArg := range args {
-			alertName, alertNamespace, crdNamespace, _, err := getInstanceInfo(false, util.AlertCRDName, util.AlertName, namespace, altArg)
+		for _, alertName := range args {
+			alertNamespace, crdNamespace, scope, err := getInstanceInfo(false, util.AlertCRDName, util.AlertName, namespace, alertName)
 			if err != nil {
+				if len(alertNamespace) == 0 && scope == apiextensions.ClusterScoped {
+					err = fmt.Errorf("%s %s doesn't appear to be running: %v", util.AlertName, alertName, err)
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_stop.go]: getInstanceInfo method will give the namespace that need to be created, it won't check if the namespace is existed or not. Either we need to check whether the namespace exist or need to check the state of the CR is running

File: pkg/synopsysctl/cmd_stop.go [hunk 2]
```diff
@@ -105,9 +110,12 @@ var stopBlackDuckCmd = &cobra.Command{
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
 		errors := []error{}
-		for _, bdArg := range args {
-			blackDuckName, blackDuckNamespace, crdNamespace, _, err := getInstanceInfo(false, util.BlackDuckCRDName, util.BlackDuckName, namespace, bdArg)
+		for _, blackDuckName := range args {
+			blackDuckNamespace, crdNamespace, scope, err := getInstanceInfo(false, util.BlackDuckCRDName, util.BlackDuckName, namespace, blackDuckName)
 			if err != nil {
+				if len(blackDuckNamespace) == 0 && scope == apiextensions.ClusterScoped {
+					err = fmt.Errorf("%s %s doesn't appear to be running: %v", util.BlackDuckName, blackDuckName, err)
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_stop.go]: getInstanceInfo method will give the namespace that need to be created, it won't check if the namespace is existed or not. Either we need to check whether the namespace exist or need to check the state of the CR is running

File: pkg/synopsysctl/cmd_stop.go [hunk 3]
```diff
@@ -154,9 +162,12 @@ var stopOpsSightCmd = &cobra.Command{
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
 		errors := []error{}
-		for _, opsArg := range args {
-			opsSightName, opsSightNamespace, crdNamespace, _, err := getInstanceInfo(false, util.OpsSightCRDName, util.OpsSightName, namespace, opsArg)
+		for _, opsSightName := range args {
+			opsSightNamespace, crdNamespace, scope, err := getInstanceInfo(false, util.OpsSightCRDName, util.OpsSightName, namespace, opsSightName)
 			if err != nil {
+				if len(opsSightNamespace) == 0 && scope == apiextensions.ClusterScoped {
+					err = fmt.Errorf("%s %s doesn't appear to be running: %v", util.OpsSightName, opsSightName, err)
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_stop.go]: getInstanceInfo method will give the namespace that need to be created, it won't check if the namespace is existed or not. Either we need to check whether the namespace exist or need to check the state of the CR is running

**Comment by rrati** [comment]
In all error cases the namespace returned is empty, making the check for empty ns redundant actually.  There really isn't a good way to improve this error message as things are implemented now.  I favor just punting on the request until later release.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/772#pullrequestreview-275407685" title="LGTM">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/772#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**